tests/test_interactive_importer.py

```diff
@@ -1,8 +1,97 @@
 import unittest
 
+from bs4 import BeautifulSoup
+
 from gourmand.importers import interactive_importer
 
 
+class TestGetImages(unittest.TestCase):
+
+    def setUp(self):
```

Comment:
This looks wrong for a generic `TestGetImages` test class. The data should probably only used in the specific test method.